packages/shared/src/components/search/SearchResult.tsx

```diff
@@ -68,11 +71,31 @@ export function SearchResult({
       },
       onSuccess: () => {
         displayToast(labels.search.feedbackText);
+
+        const eventNames = {
+          1: AnalyticsEvent.UpvoteSearch,
+          [-1]: AnalyticsEvent.DownvoteSearch,
+        };
+
+        trackEvent({
+          event_name: eventNames[chunk.feedback],
+          target_id: chunk.id,
+        });
```

Comment:
I think we need to move this to the onMutate, it's important to know regardless of it suceeding or not.

Assumption for now.
Tagging @idoshamun to verify that was the case for tracking.

Reply:
moved to `onMutate` for now, let me know if I need to rollback

Reply:
It's ok :)